opflexagent/gbp_ovs_agent.py

```diff
@@ -35,7 +35,6 @@
 from opflexagent import as_metadata_manager
 from opflexagent import constants as ofcst
 from opflexagent import rpc
-from opflexagent import snat_iptables_manager
```

Comment:
It was still breaking the UTs, and I misunderstood the reason why this was happening. I've posted a patch in master that should fix it